dali/dfuplus/dfuplus.cpp

```diff
@@ -30,6 +30,7 @@
 
 #ifdef DAFILESRV_LOCAL
 #include "rmtfile.hpp"
+#include "rmtclient.hpp"
 #include "sockfile.hpp"
```

Comment:
I thought sockfile.hpp was deleted?

Reply:
no, it remains the main dafilesrv server component and in lib dafsserver.
I considered renaming it, feeling I prob. still should